jwst/outlier_detection/tests/test_outlier_detection.py

```diff
@@ -514,7 +514,9 @@ def test_outlier_step_on_disk(three_sci_as_asn, tmp_cwd):
         result_files
     ) + len(blot_files)
 
-
+@pytest.mark.xfail(
+    reason="random data are flagged as CR; pixel area set to 1 srad in meta."
```

Comment:
It's actually not the random data that are flagged, it's the square added to it.  Maybe this, for clarity?
```suggestion
    reason="Test data needs to be fixed to avoid outliers being detected."
```